docs/installation/k9s-installation.md

```
@@ -0,0 +1,133 @@
+# Install K9s Plugin
+
+Integrate HolmesGPT into your [K9s](https://github.com/derailed/k9s){:target="\_blank"} Kubernetes terminal for instant analysis.
+
+![K9s Demo](../assets/K9sDemo.gif)
+
+### Prerequisites
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)